<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 128, 128);</string>
  </property>
  <widget class="QWidget" name="rootPanel">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 128, 128);</string>
   </property>
   <widget class="QFrame" name="salesPanel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1151</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="adminPanel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1151</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="adminContent">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1151</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="adminLogin" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>1151</width>
        <height>781</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1151</width>
        <height>781</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 128, 128);</string>
      </property>
      <widget class="QLineEdit" name="usernameInput">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>310</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 102, 102);
border-radius: 5px;
color: rgb(102, 178, 178);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordInput">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>360</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 102, 102);
border-radius: 5px;
color: rgb(102, 178, 178);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginBtn">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>410</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(178, 216, 216);
	border-radius: 5px;
	color: rgb(0, 76, 76);
	font: 87 12pt &quot;Segoe UI Black&quot;;
}

QPushButton:hover {
	background-color: rgb(0, 76, 76);
	border-radius: 5px;
	color: rgb(178, 216, 216);
	font: 87 12pt &quot;Segoe UI Black&quot;;
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>969</x>
         <y>30</y>
         <width>161</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 76, 76);
font: 87 12pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="homeBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
	background-color: rgb(0, 76, 76);
	color: rgb(178, 216, 216);
}

QPushButton:hover {
	border: none;
	background-color: rgb(0, 128, 128);
	color: rgb(178, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>710</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	background-color: rgb(0, 76, 76);
	color: rgb(178, 216, 216);
}

QPushButton:hover {
	border: none;
	background-color: rgb(0, 128, 128);
	color: rgb(178, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="salesBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	background-color: rgb(0, 76, 76);
	color: rgb(178, 216, 216);
}

QPushButton:hover {
	border: none;
	background-color: rgb(0, 128, 128);
	color: rgb(178, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>Sales</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: none;
	background-color: rgb(0, 76, 76);
	color: rgb(178, 216, 216);
}

QPushButton:hover {
	border: none;
	background-color: rgb(0, 128, 128);
	color: rgb(178, 216, 216);
}</string>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="home">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>11</y>
       <width>1151</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="admin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>1151</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QFrame" name="sales">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>1151</width>
       <height>781</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: none;
	background-color: rgb(0, 76, 76);
	color: rgb(178, 216, 216);
}

QPushButton:hover {
	border: none;
	background-color: rgb(0, 128, 128);
	color: rgb(178, 216, 216);
}</string>
     </property>
     <property name="text">
      <string>Sales</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="homePanel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1151</width>
      <height>781</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 216, 216);
border-radius: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>salesPanel</zorder>
   <zorder>sidebar</zorder>
   <zorder>homePanel</zorder>
   <zorder>adminPanel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>homeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayHome()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adminBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>userAuthenticate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>1279</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>-39</x>
     <y>751</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>salesBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displaySales()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adminBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>userAuthenticate()</slot>
  <slot>displayHome()</slot>
  <slot>displaySales()</slot>
  <slot>displayAdmin()</slot>
 </slots>
</ui>
